File: 1848-sum-of-unique-elements/sum-of-unique-elements.java
class Solution {
    public int sumOfUnique(int[] nums) {
        HashMap<Integer, Integer> map = new HashMap<>();

        for(int ele : nums){
            if(map.containsKey(ele)){
                int freq = map.get(ele);
                map.put(ele, freq+1);
            }
            else{
                map.put(ele, 1);
            }
        }

        int sum = 0;

        for(int key : map.keySet()){
            if(map.get(key) == 1){
                sum += key;
            }
        }
        return sum;  
    }
}

File: 1848-sum-of-unique-elements/sum-of-unique-elements.py
class Solution(object):
    def sumOfUnique(self, nums):
        map = {}
        for ele in nums:
            if ele in map:
                map[ele] += 1
            else:
                map[ele] = 1

        sum = 0
        for key in map.keys():
            if map[key] == 1:
                sum += key
        return sum